### PR TITLE
Internal: Adjust event schema [ED-13547]

### PR DESCRIPTION
**Pull request by murzin-nikita**
## PR Checklist

- [ ] The commit message follows our guidelines:  https://github.com/elementor/elementor/blob/master/.github/CONTRIBUTING.md

## PR Type
What kind of change does this PR introduce?

- [ ] Bugfix
- [ ] Feature
- [ ] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] CI related changes
- [ ] Documentation content changes
- [ ] Other... Please describe:

## Summary

This PR can be summarized in the following changelog entry:

*

## Description
An explanation of what is done in this PR

*

## Test instructions
This PR can be tested by following these steps:

*

## Quality assurance

- [ ] I have tested this code to the best of my abilities
- [ ] I have added unittests to verify the code works as intended
- [ ] Docs have been added / updated (for bug fixes / features)

Fixes #

### CODE REVIEW

File: modules/editor-events/assets/js/editor/event.js [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+		this.metadata = {
+			type: 'editor_events',
+			action: 'update',
+			time,
+			schema_id: 1,
+			version: 1,
+			publisher_version: elementor.config.editor_events?.elementor_version,
+			guid: uuidv4(),
+		};
+	}
```
Comment (louiswol94) [inline modules/editor-events/assets/js/editor/event.js:41]: @mykytamurzin Where does `this.metadata` ever get used?

File: modules/editor-events/assets/js/editor/module.js [hunk 2]
```diff
@@ -1,12 +1,21 @@
...
+	sections = {
+		topbar: 'topbar',
+	};
+
+	elementNames = {
+		siteSettings: 'site_settings',
+	};
+
+	types = {
+		button: 'button',
```
Comment (louiswol94) [inline modules/editor-events/assets/js/editor/module.js:18]: Maybe these consts should be in a separate file?

File: modules/editor-events/module.php [hunk 3]
```diff
@@ -20,6 +22,13 @@ public static function get_editor_events_config() {
 
 		$settings = [
 			'can_send_events' => $can_send_events,
+			'elementor_version' => ELEMENTOR_VERSION,
+			'site_url' => get_site_url(),
+			'wp_version' => get_bloginfo( 'version' ),
+			'user_agent' => esc_html( Utils::get_super_global_value( $_SERVER, 'HTTP_USER_AGENT' ) ),
+			'site_language' => get_locale(),
```
Comment (louiswol94) [inline modules/editor-events/module.php:29]:
```suggestion
			'site_language' => get_bloginfo( 'language' ),
```
Which one is better?

Comment (louiswol94) [reply modules/editor-events/module.php:29]: Justa. question, get_locale() might be better :)

File: modules/editor-events/assets/js/editor/event.js (hunk 1 above)
Comment (mykytamurzin) [reply modules/editor-events/assets/js/editor/event.js:41]: it's not used directly, only when I convert event object to json to get necessary structure:
```
{
    metadata: {..},
    payload: {...},
}
```
btw I have the same for payload (`this.payload`)

File: modules/editor-events/module.php (hunk 3 above)
Comment (mykytamurzin) [reply modules/editor-events/module.php:29]: I checked what we are already using for System Info page and just re-used most of functions for this. The main difference is: 
`get_bloginfo( 'language' )` - will return 'en-US'
`get_locale()` - will return `en_US`

not a big difference actually :)

**Review by louiswol94 — APPROVED** [review]

**Review by davseve — APPROVED** [review]

**Pull request merged**